projects/plugins/jetpack/modules/shortcodes/unavailable.php

```diff
@@ -70,6 +70,8 @@ public function stub_shortcode( $atts, $content = '', $shortcode = '' ) {
 function jetpack_init_shortcode_unavailable() {
 	new Jetpack_Shortcode_Unavailable(
 		array(
+			'hulu'                    => __( 'Hulu no longer allows embedding', 'jetpack' ),
+			'ustream'                 => __( 'Ustream has been replaced with IBM video', 'jetpack' ),
```

Comment:
Is this correct?

```suggestion
			'hulu'                    => __( 'Hulu no longer allows embedding.', 'jetpack' ),
			'ustream'                 => __( 'Ustream has been replaced with IBM video, and no longer allows embedding.', 'jetpack' ),
```

Reply:
Great question... I thought that was the case due to the fact that it forwards to IBM **and** because WordPress.com removed it from the support docs. However, as I dig a little deeper it still is a working service. I do see that the shortcode is not working though because the embed URL structure has slightly changed.